sandbox/controls/ListItems.qml

```diff
@@ -331,12 +331,11 @@ CExPynn1gWf9bx498P7/nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2I
     StatusListItem {
         title: "List Item with Tags"
         icon.isLetterIdenticon: true
-        tags: [
-            StatusListItemTag {
-                title: qsTr("Tag 1")
-                icon.isLetterIdenticon: true
-            }
-        ]
+        tagsModel: 3
```

Comment:
Shouldn't model be a `ListModel` here instead of a number?

Reply:
model also accept number as per the docs